src/openms/include/OpenMS/ANALYSIS/OPENSWATH/MRMDecoy.h

```diff
@@ -216,6 +224,11 @@ namespace OpenMS
     OpenMS::TargetedExperiment::Peptide reversePeptide_(
       const OpenMS::TargetedExperiment::Peptide& peptide) const;
 
+    /**
+     @breif Convert a peptide to a string which contains the peptide sequence and modifications
```

Comment:
```suggestion
     @brief Convert a peptide to a string which contains the peptide sequence and modifications
```

Reply:
can you also document in which format it is? is this the default AASequence we use in OpenMS

Reply:
> Btw you can add suggestions to a batch so that only one commit is created. Each commit starts a CI run of which we only have a limited amount per month.

Sorry and thanks for letting me know. I will do that next time.